test/functional-vanilla.bats

```diff
@@ -151,6 +151,43 @@ spec:
   done
 }
 
+@test "Rename Agents" {
+  initAgents
+  echo "========> Config.yaml before rename"
+  cat ~/.iofog/config.yaml
+  for IDX in "${!AGENTS[@]}"; do
+    local AGENT_NAME="${NAME}-${IDX}"
+    test iofogctl -v -n "$NS2" rename agent "$AGENT_NAME" "${AGENT_NAME}-test_rename"
+    echo "========> Config.yaml"
+    cat ~/.iofog/config.yaml
+    test iofogctl -v -n "$NS2" logs agent "${AGENT_NAME}-test_rename"
+    checkLegacyAgent "${AGENT_NAME}-test_rename" "$NS2"
+    test iofogctl -v -n "$NS2" rename agent "${AGENT_NAME}-test_rename" "${AGENT_NAME}"
+  done
+}
+
+@test "Rename Controller" {
+  initVanillaController
+  test iofogctl -v -n "$NS2" rename controller "$NAME" "${NAME}-newname"
+  test iofogctl -v -n "$NS2" logs controller "${NAME}-newname"
+  test iofogctl -v -n "$NS2" rename controller "${NAME}-newname" "$NAME"
+  test iofogctl -v -n "$NS2" logs controller "${NAME}"
+  checkLegacyController "$NS2"
+}
+
+@test "Rename Connector" {
+  initVanillaController
+  test iofogctl -v -n "$NS2" rename connector "$NAME" "${NAME}-newname"
+  test iofogctl -v -n "$NS2" rename connector "${NAME}-newname" "$NAME"
+}
```

Comment:
This only tests that the iofogctl command does not fail ( a.k.a the CLi command returns an exit code of 0). It does not test that the connector has actually been renamed. Nor does it test if there is any artifact of the old connector name left